app/views/doorkeeper_applications/edit.html.erb

```diff
@@ -23,6 +23,14 @@
   doorkeeper_application_supported_permissions_path(@application),
   class: "govuk-link" %>
 
+<%= link_to "View access log",
```

Comment:
Not a big deal but this could probably have come in the last commit that added the access log page.

Reply:
Yep, I'll rebase that in.